app/src/main/java/com/isc/hermes/database/Hermesdb.java

```diff
@@ -0,0 +1,45 @@
+package com.isc.hermes.database;
+
+import com.mongodb.client.MongoDatabase;
+
+/**
+ * This class is responsible for generating the connection to the hermes database.
+ */
+public class Hermesdb {
+
+    private Connection connection;
+    private MongoDatabase database;
+    private static Hermesdb hermesdb;
+    private final String DATABASE_NAME = "hermesmapdb";;
```

Comment:
You have a ";" too much, it would be good if you delete it.

Reply:
done